defaults/config.js

```diff
@@ -1,510 +1,428 @@
-		//
 		username: "thelounge",
-
-		//
-		// Real Name
-		//
-		// @type     string
-		// @default  "The Lounge User"
-		//
 		realname: "The Lounge User",
-
-		//
-		// Channels
-		// This is a comma-separated list.
-		//
-		// @type     string
-		// @default  "#thelounge"
-		//
 		join: "#thelounge",
 	},
 
+	// ### `displayNetwork`
 	//
-	// Set socket.io transports
+	// When set to `false`, network fields will not be shown in the "Connect"
+	// window.
 	//
-	// @type     array
-	// @default  ["polling", "websocket"]
+	// Note that even though users cannot access and set these fields, they can
+	// still connect to other networks using the `/connect` command. See the
+	// `lockNetwork` setting to restrict users from connecting to other networks.
 	//
-	transports: ["polling", "websocket"],
+	// This value is set to `true` by default.
+	displayNetwork: true,
 
+	// ### `lockNetwork`
 	//
-	// Run The Lounge using encrypted HTTP/2.
-	// This will fallback to regular HTTPS if HTTP/2 is not supported.
+	// When set to `true`, users will not be able to modify host, port and TLS
+	// settings and will be limited to the configured network.
 	//
-	// @type     object
-	// @default  {}
+	// It is often useful to use it with `displayNetwork` when setting The
+	// Lounge as a public web client for a specific IRC network.
 	//
-	https: {
-		//
-		// Enable HTTP/2 / HTTPS support.
-		//
-		// @type     boolean
-		// @default  false
-		//
-		enable: false,
+	// This value is set to `false` by default.
+	lockNetwork: false,
 
-		//
-		// Path to the key.
-		//
-		// @type     string
-		// @example  "sslcert/key.pem"
-		// @default  ""
-		//
-		key: "",
+	// ## User management
 
-		//
-		// Path to the certificate.
-		//
-		// @type     string
-		// @example  "sslcert/key-cert.pem"
-		// @default  ""
-		//
-		certificate: "",
+	// ### `messageStorage`
 
-		//
-		// Path to the CA bundle.
-		//
-		// @type     string
-		// @example  "sslcert/bundle.pem"
-		// @default  ""
-		//
-		ca: "",
-	},
+	// The Lounge can log user messages, for example to access them later or to
+	// reload messages on server restart.
 
+	// Set this array with one or multiple values to enable logging:
+	// - `text`: Messages per network and channel will be stored as text files.
+	//   **Messages will not be reloaded on restart.**
+	// - `sqlite`: Messages are stored in SQLite database files, one per user.
 	//
-	// Default quit and part message if none is provided.
+	// Logging can be disabled globally by setting this value to an empty array
+	// `[]`. Logging is also controlled per user individually in the `logs` key of
+	// their JSON configuration file.
 	//
-	// @type     string
-	// @default  "The Lounge - https://thelounge.chat"
+	// This value is set to `["sqlite", "text"]` by default.
+	messageStorage: ["sqlite", "text"],
+
+	// ### `useHexIp`
 	//
-	leaveMessage: "The Lounge - https://thelounge.chat",
+	// When set to `true`, users' IP addresses will encoded has hex.
```

Comment:
`has` -> `as`

And specify that this is encodes it in the `username` field and it only supports IPv4.

Reply:
Done, let me know what you think.